docs/sts-migrate-to-private-bucket.md

```diff
@@ -0,0 +1,428 @@
+# AWS Security Token Service - Steps to Migrate the OIDC issuer from public S3 Bucket to CloudFront Distribution
+
+> NOTE: the steps described in this document are not supported or recommended to be done in a production environment.
+
+- [Prerequisites](#Prerequisites)
+- [Test the existing token](#test-token)
+- [Setup new OIDC with CloudFront Distribution](#setup)
+    - [Create the CloudFront Distribution](#setup-cloudfront)
+    - [Create and patch the new OIDC discovery documents and JWKS](#setup-oidc-documents)
+    - [Create the new OIDC using CloudFront Distribution](#setup-oidc-idp)
+- [Backing up current cluster data](#backup)
+- [Patch the cluster to use the new OIDC](#patch-cluster)
+- [Revoke public access to S3 Bucket](#revoke-s3-public-access)
+- [Rollback to OIDC with S3 Public URL](#rollback)
+- [Rollback to OIDC with S3 Public URL](#delete)
+
+## Prerequisites <a name="Prerequisites"></a>
+
+- An OCP Cluster created on AWS with manual authentication mode with STS using S3 Bucket as OIDC URL (issuerURL)
```

Comment:
"manual authentication mode" changed to "authentication mode".
In my understanding, manual means the component creds created manually.